test/functional/ui/output_spec.lua

```diff
@@ -22,6 +22,7 @@ describe("shell command :!", function()
 
   after_each(function()
     screen:detach()
+    session.command('call jobstop(b:terminal_job_id)')
   end)
 
   it("displays output even without LF/EOF. #4646 #4569 #3772", function()
```

Comment:
I just noticed that https://github.com/jamessan/neovim/commit/f332eba16c290c729b21171ed83b1e3377ee17ec made this test pointless. :<